mitsuba-blender/io/exporter/camera.py

```diff
@@ -25,6 +25,9 @@ def export_camera(camera_instance, b_scene, export_ctx):
     else:
         export_ctx.log(f'Unknown \'sensor_fit\' value when exporting camera: {sensor_fit}', 'ERROR')
 
+    params["principal_point_offset_x"] = b_camera.data.shift_x
+    params["principal_point_offset_y"] = -b_camera.data.shift_y / res_x * res_y
```

Comment:
This does not seem to work for non-square images. I don't know what the rationale is for dividing by the aspect ratio. Can you elaborate ?

You can find a proof-of-concept for the failing case below.
[bug_poc.zip](https://github.com/mitsuba-renderer/mitsuba-blender/files/9463849/bug_poc.zip)

Reply:
Aww, I must've tested it for the vertical case only, thank you for checking! Re-implemented according to [this one](https://blender.stackexchange.com/questions/58235/what-are-the-units-for-camera-shift). Seems to be working fine for both cases.

Reply:
Previously thought that the shift is measured in res_x units for both axes

Reply:
Seems to work fine!